src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -638,13 +643,18 @@ public void Initialize()
 
             options.OnRecoveryError += (obj, e) =>
             {
+                string title = "Recovery Error - System Storage";
+
+                if (Logger.IsOperationsEnabled)
+                    Logger.Operations($"{title}. {e.Message}", e.Exception);
+
                 var alert = AlertRaised.Create(
                     null,
-                    "Recovery Error - System Storage",
+                    title,
                     e.Message,
                     AlertType.RecoveryError,
                     NotificationSeverity.Error,
-                    "Recovery Error System");
+                    key: $"Recovery Error System/{SystemTime.UtcNow.Ticks % 5}"); // if this was called multiple times let's try to not overwrite previous alerts
```

Comment:
That give it at a very high granularity, no? 
Shouldn't we use something like `1 min` interval, at least?

If this is called 30 times a second, for example , we'll spam the user

Reply:
Note this is for System storage environment only. We'll have max 5 alerts

Reply:
Oh, I see. Missed that there is a total of 5.